src/main/java/com/uwetrottmann/tmdb2/TmdbInterceptor.java

```diff
@@ -26,9 +53,13 @@ public Response intercept(Chain chain) throws IOException {
 
     /**
      * If the host matches {@link Tmdb#API_HOST} adds a query parameter with the API key.
+     *
+     * @throws TmdbUnathorizedException          If user try to access a resource which requires authenticate, without a session.
+     * @throws TmdbAuthenticaitonFailedException
      */
-    public static Response handleIntercept(Chain chain, String apiKey) throws IOException {
```

Comment:
i can't push a single file, there are many changes, that will break the code as it is.